common/src/cobs/patch.rs

```diff
@@ -476,6 +486,8 @@ pub struct Revision {
     pub merges: Vec<Merge>,
     /// When this revision was created.
     pub timestamp: Timestamp,
+    /// Changeset of this revision.
+    pub changeset: Option<Diff>,
```

Comment:
What's going on here? Should this be in another PR? Looks like it's just set to `None`

Reply:
Yeah seems like the PRs got mixed up, and this leaked into this PR. Will separate it today